fourInRowJava.java

```diff
@@ -0,0 +1,126 @@
+
+public class Game 
+    {
+        
+        public static boolean firstPlayer(player p)
+        {
+            for (int col = 0; col < COL; col++)
+            {
+                for (int row = 0; row < ROW; row++)
+                {
+                    if (col == p1.Col() && row == p1.Row())
+                    {
+                        board[col][row] = "+";
+                    }
+                }
+            }
+        }
+        public static boolean secondPlayer(player p)
+        {
+            for (int col = 0; col < COL; col++)
+            {
+                for (int row = 0; row < ROW; row++)
+                {
+                    if (col == p1.Col() && row == p1.Row())
+                    {
+                        board[col][row] = "-";
+                    }
+                }
+            }
+        }
+    }
+    public interface class board
+    {
+        final static int row = 7;
+        final static int col = 6;
+        char[][] TheBoard = {
+                                        {" ", " ", " ", " ", " ", " ", " "}, 
+                                        {" ", " ", " ", " ", " ", " ", " "}, 
+                                        {" ", " ", " ", " ", " ", " ", " "}, 
+                                        {" ", " ", " ", " ", " ", " ", " "}, 
+                                        {" ", " ", " ", " ", " ", " ", " "}, 
+                                        {" ", " ", " ", " ", " ", " ", " "}};
+        
+        public static void printBoard(board b)
+        {
+            for (int col = 0; col < COL; col++)
+            {
+                for (int row = 0; row < ROW; row++)
+                {
+                    System.out.println(TheBoard[col][row] + "");    
+                }
+            }
+        }
+    }
+    
+    class player
+    {
+        static int Row;
+        static int Col;
+    }
```

Comment:
# I'm sorry but things wrong with this class:
* no accesses modifier for the class, it is decaled as default but it will be nice if you write it just for clarity.
* no constructor, making it impossible to to decide what are the starting values of the object.
* again no accesses modifiers for the variables, it is decaled as default but it will be nice if you write it just for clarity.
* the variables are static making them the same between all the objects, change one of them, u change them all.
* and finally, I am sorry if I am to harsh but what is even the porous of this class? It has no internal logic and it looks like it is used in different ways across the project.